web/focus/focusconnect.py

```diff
@@ -119,6 +119,18 @@ def uitkeringsspecificaties(self, bsn, url_root):
             uitkeringsspecificaties = convert_uitkeringsspecificaties(xml_uitkeringspec, url_root)
             return uitkeringsspecificaties
 
+    def EAanvragenTozo(self, bsn, url_root):
+        with self._client.options(raw_response=True):
+            raw_tozo_documente = self._client.service.getEAanvraagTOZO(bsn=bsn).content.decode("utf-8").replace("\n", "")
+            result = re.search(r"<return>.*<\/return>", raw_tozo_documente)
+            if not result:
+                # This can return something else apparently. Lets log this so we can debug this.
+                logger.error("no body getEAanvraagTOZO? %s" % raw_tozo_documente)
```

Comment:
Niet privacygevoelig?

Reply:
Dit gebeurd wanneer er geen soap antwoord komt. Hier hoort geen gevoelige informatie in te staan.